### ISSUE TITLE
Grab the entire remix tree

### ISSUE THREAD

**Issue by cycomachead** [snap-cloud/snapCloud#175]
Lower priority. 
Getting remixes should load the entire tree.

**Comment by bromagosa** [comment]
Hmm, I wouldn't say this is lower priority. Collections come before this, but I think this should come right after that :)

**Comment by cycomachead** [comment]
This is annoying to figure out how to do well. At least with the current structure. It might make sense to add a 'root_project_id' to the remixes table. Since it seems easier to go backwards up a tree than down (at least in SQL).

**Comment by cycomachead** [comment]
Never mind! I figured it out with some googling and the always helpful postgres wiki

https://snap-analytics.cs10.org/question/124

**Comment by cycomachead** [comment]
Yeah. That makes sense. The inverse is a bit easier to construct. I’ll add a method for that when I handle getting the tree.
